### DISCUSSION TITLE
Am I running at 500hz packet rate?

### DISCUSSION THREAD

**Discussion by TwoToneEddy** [ExpressLRS/ExpressLRS#1490] [Q&A]
Hi,
I understand the header in the ELRS lua script will tell me how many good and bad packets I'm getting, ie, 0/500 means I'm running at 500hz and getting 0 bad packets. For some reason my header is showing 0/250 no matter what I do. If I go to say 50hz I would expect to see 0/50 wouldn't I? Am I not understanding something? I'm using a tango 2 with the happy model slim pro 2.4ghz
Thanks
Lee

**Comment by TwoToneEddy** [comment]
Does anyone know what's going on here or should I get a scope on it?

**Reply by StonedDawg** [reply]
this means your handset is running just CRSF, not CRSFSHOT (with mixer sync). you need to update your handset firmware

**Reply by TwoToneEddy** [reply]
Thanks for the information, it doesn't seem to go lower than 250 either though so I don't think that's the issue.

**Comment by pkendall64** [comment]
If it stays at 0/250 even if you change the packet rate then either the radio does not have crsfshot or you've compiled the firmware with the OpenTX sync option disabled.

**Reply by TwoToneEddy** [reply]
Thanks for the reply, it's odd that it also doesnt change from 0/250 even when I go to a lower rate though.

**Reply by pkendall64** [reply]
Without CRSFSHOT the radio will shove the packets at the module as fast as it can handle. The module is reporting how many packets it can see in the measure interval (i.e. 1 second) thats why even though you configure 50Hz, you see 0/250. The module will just throw away 4 of the 5 packets and send the latest one out the RF link to the receiver.

**Reply by TwoToneEddy** [reply]
Ah ok, that makes sense. I thought the tango 2 on the latest version of freedomtx had crsfshot but maybe not. Thanks for clarifying this.

**Reply by TwoToneEddy** [reply]
![PXL_20220321_184319915.jpg](https://user-images.githubusercontent.com/18442293/159342475-79a873ba-e6ed-493c-98d2-3e4d4651a513.jpg)

Looks like I'm using crsfshot

**Reply by pkendall64** [reply]
Only thing I can think is that you disabled the FEATURE_OPENTX_SYNC option in configurator when building and flashing the module.

**Reply by TwoToneEddy** [reply]
Doesn't look like that's even an option for 2.3.

**Reply by TwoToneEddy** [reply]
Oh hang on, seems to have appeared now. I'm sure it wasn't there a second ago. This is enabled though. Thanks for the idea

**Reply by TwoToneEddy** [reply]
Looks like that feature was deprecated since ver 2.2.0 so I'm not sure why my configurator is showing it.

**Comment by vanawaker** [comment]
have you solved this? I'm occurring the same issue, tango2 with betafpv elrs nano module, no matter what packet rate I'm setting in Lua script, the top bar stack at 0:250, Its in CRSFshot, and I've tried frsky xlitepro, it's working fine no issue, I'm wondering maybe the TBS banned the ELRS module on tango2 or at least limited some functions like lock the packet rate?

**Reply by sweebee** [reply]
Tango 2 can’t go higher than 250hz. A limitation of FreedomTX.

**Reply by vanawaker** [reply]
I know it couldn't higher than 250hz, it is not adjustable, no matter I set it to 150hz, or 50hz, it is still stuck at 0/250, seems like not communicate with module

**Reply by TwoToneEddy** [reply]
I am still seeing this issue, stuck at 250hz, I can't got faster or slower.
